java/build-android.gradle

```diff
@@ -24,7 +26,11 @@ project.buildDir = buildDir
 project.version = rootProject.file('../VERSION_NUMBER').text.trim()
 project.group = "com.microsoft.onnxruntime"
 
-def mavenArtifactId = project.name + '-mobile'
+def mavenArtifactId = isMobileBuild ? project.name + '-mobile' : project.name
+def mobileDescription = 'ONNX Runtime Mobile package is a size optimized inference library for executing ' +
+	'ONNX (Open Neural Network Exchange) models. This package is built from the open source ' +
+	'inference engine targeting reduced disk footprint for mobile platforms.'
+def defaultDescription = 'Manash Fill this'
 
```

Comment:
On Maven we currently have 'onnxruntime' and 'onnxruntime_gpu' which are ORT + Java.

onnxruntime-mobile is the existing mobile package.

Some random person has added onnxruntime-android-amd64 for ORT 1.7

I'm leaning towards either onnxruntime-android for the full package with the android AAR. That keeps the usage of 'onnxruntime-mobile' consistent across android and ios packages. 

The description for the new package needs to be very clear about what it is, and ideally we can update the description of the 'onnxruntime' and 'onnxruntime_gpu' packages to call out that they're ORT + Java for win/osx/linux platforms.

Reply:
per mobile scrum change name to `onnxruntime-anrdoid`. This is consistent with PyTorch, Tensorflow package names.